### ISSUE TITLE
about the code

### ISSUE THREAD

**Issue by sunxia233** [vana77/Bottom-up-Clustering-Person-Re-identification#16]
thank you for your sharing ,I don't quite understand some places in the code.
Firstly，
dists.add_(torch.tril(100000 * torch.ones(len(u_feas), len(u_feas)))) 
Why does this distance matrix need to add a lower triangular matrix with a content of 10000?
Secondly，
 for idx in range(len(u_feas)):
            for j in range(idx + 1, len(u_feas)):
                if label[idx] == label[j]:
                    dists[idx, j] = 100000
What is the meaning of this distance matrix loop?

I really didn't understand it, so I asked you questions, thank you very much and look forward to your reply.
